src/main/java/seedu/duke/parsers/AddTaskParser.java

```diff
@@ -21,7 +21,7 @@
     private static final String TASK_ESTIMATED_WORKING_TIME_STR = StringConstants.TASK_ESTIMATED_WORKING_TIME_STR;
     private static final String TASK_NAME = StringConstants.TASK_NAME;
     private static final String TASK_DESCRIPTION = StringConstants.TASK_DESCRIPTION;
-    private static final String TASK_WORKING_TIME = StringConstants.TASK_ESTIMATED_WORKING_TIME;
+    private static final String TASK_ESTIMATED_WORKING_TIME = StringConstants.TASK_ESTIMATED_WORKING_TIME;
```

Comment:
I suggest just use the constants from StringConstants directly without repeated declaration.

Reply:
This is to be consistent. We cant have a `StringConstants.TASK_ESTIMATED_WORKING_TIME` for eg within our methods, right?